packages/store/gas-report.json

```diff
@@ -363,19 +363,19 @@
     "file": "test/Mixed.t.sol",
     "test": "testRegisterAndGetFieldLayout",
     "name": "register Mixed table",
-    "gasUsed": 582081
+    "gasUsed": 582351
   },
   {
     "file": "test/Mixed.t.sol",
     "test": "testSetAndGet",
     "name": "set record in Mixed",
-    "gasUsed": 103978
+    "gasUsed": 104908
```

Comment:
gas only increased around 1k for set record here (i expected 2k for the additional sload, but seems like some of it was offset by less memory being passed around between functions and the optimized `getFieldLayout`)